ateam_kenobi/src/double_touch_eval.hpp

```diff
@@ -101,13 +104,13 @@ class DoubleTouchEval
   std::optional<Robot> GetRobotTouchingBall(const World & world)
   {
     auto found_iter = std::ranges::find_if(
-      world.our_robots, [this, &world](const std::optional<Robot> & maybe_robot) {
-        if (!maybe_robot) {
+      world.our_robots, [this, &world](const Robot & robot) {
+        if (!robot.visible) {
```

Comment:
I like the use of the visible robots helpers instead of std::optional for these checks :)